packages/rrweb/test/__snapshots__/integration.test.ts.snap

```diff
@@ -6164,6 +6164,23 @@ exports[`record integration tests should mask text in form elements 1`] = `
         "top": 0
       }
     }
+  },
+  {
+    "type": 3,
+    "data": {
+      "source": 0,
+      "texts": [],
+      "attributes": [
+        {
+          "id": 87,
+          "attributes": {
+            "value": "*** *****"
```

Comment:
this was unmasked before the change, triggered by `el.setAttribute('value', 'new value')`.